### PR TITLE
Adding Live Interpretation parameter, and matching syntax for String

### PR DESCRIPTION
**Pull request by bwilsonms**
Adding new parameter for Live Interpretation to Set-CsTeamsMeetingPolicy, matching all string to String

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -258,6 +258,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LiveInterpretationEnabledType
+Allows meeting organizers to configure a meeting for Language Interpretation, selecting attendees of the meeting to become interpreters that other attendees can select and listen to the real-time translation they provide.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:262]:
```suggestion
Allows meeting organizers to configure a meeting for language interpretation, selecting attendees of the meeting to become interpreters that other attendees can select and listen to the real-time translation they provide.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -258,6 +258,24 @@ Accept pipeline input: False
...
+Possible values are:
+- **DisabledUserOverride**, if you would like users to be able to use interpretation in meetings but by default it is disabled. 
+- **Disabled**, prevents the option to be enabled in Meeting Options
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:265]:
```suggestion
- **Disabled**, prevents the option to be enabled in Meeting Options.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bwilsonms, kindly commit the minor suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]
I'm fine after Johan's suggestions.

**Comment by dariomws** [comment]
@bwilsonms, will this parameter be also added to New-CsTeamsMeetingPolicy?

**Review by get-itips — APPROVED** [review]

**Comment by bwilsonms** [comment]
> Copy review - @bwilsonms, kindly commit the minor suggested changes. Many thanks.

Done, thanks for the feedback.

**Comment by bwilsonms** [comment]
> @bwilsonms, will this parameter be also added to New-CsTeamsMeetingPolicy?

Possibly, the developer only suggested here. Will ask and add to New as well if needed.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**